kuma/javascript/src/breadcrumbs.jsx

```diff
@@ -8,6 +11,22 @@ type DocumentProps = {
 };
 
 export default function Breadcrumbs({ document }: DocumentProps) {
+    const ga = useContext(GAProvider.context);
+
+    /**
+     * Send a signal to GA when there is an interaction on one
+     * of the breadcrumb menu links.
+     * @param {String} label  - The label for the GA event
+     */
+    const sendBreadcrumbItemClick = label => {
```

Comment:
I think we should do the same here. Instead of creating a new function, we should just do:
```javascript
function sendBreadcrumbItemClick(event) {
  const label = event.target.href;
  ga('send', ...)
}  
```